<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagePatcherUI</class>
 <widget class="QWidget" name="ImagePatcherUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Patcher UI by JMDigital</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagepatcherui.qrc">
    <normaloff>:/images/images/imagepatcherui.ico</normaloff>:/images/images/imagepatcherui.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="sourceLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>430</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse for a folder that contains your highest resolution images. Note: Images MUST be larger than your desired Patch Size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>This is your input folder containing all the images you want to process</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>430</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>This is your output folder containing the full size patched images</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowResOutputLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>430</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>This is your output folder containing the scaled Low-Res size patched images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseSourceButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseOutputButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseLowResOutputButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSourceFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Source Folder:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutputFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output Folder:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutputLowResFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output Low-Res Folder:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPatchSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Patch Size:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLowResScale">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Low-Res Scale:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxScale">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>2x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8x</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="patchSizeText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>e.g 512 (without px)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="processButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>457</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>true</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Image Patcher UI</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>568</y>
     <width>600</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>8</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imagepatcherui.qrc">
     <normaloff>:/images/images/JMDigital.png</normaloff>:/images/images/JMDigital.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>46</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="progressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagepatcherui.qrc"/>
 </resources>
 <connections/>
</ui>
